connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -272,5 +273,13 @@ message StatFunction {
   message Summary {
     repeated string statistics = 1;
   }
+
+  // StatFunctions.crossTabulate
+  message CrossTabulate {
```

Comment:
Can we add a bit more comments on such proto? Basically proto is also API that someone will read to understand the contract.

Beam's implementation offers a good example of comments for proto: https://github.com/apache/beam/blob/master/model/pipeline/src/main/proto/org/apache/beam/model/pipeline/v1/beam_runner_api.proto

Reply:
I am thinking for such function we can document
1. what this function does
2. its parameters (e.g. range, examples, etc.)
3. some caveats (invalid usage, etc.)

Reply:
nice, will update